### PR TITLE
Fixes sanitize_simple() to work

### PR DESCRIPTION
**Pull request by Tastyfish**
- `sanitize_simple()`... wasn't actually doing anything before.

### CODE REVIEW

File: code/__HELPERS/bygex/bygex.dm [hunk 1]
```diff
@@ -48,11 +48,13 @@
 	regex_replaceall(str, exp, fmt)
 		return call(LIBREGEX_LIBRARY, "regex_replaceall")(str, exp, fmt)
 
+#if DM_VERSION < 510
 	replacetextEx(str, exp, fmt)
```
Comment (tigercat2000) [inline code/__HELPERS/bygex/bygex.dm]: this only covers the bygex replacetext, you should find the old one too

Comment (Tastyfish) [inline code/__HELPERS/bygex/bygex.dm]: Not sure if the game works if you turn off BYGEX and use the old procs, but done nevertheless.

**Comment by Fox-McCloud** [comment]
related: https://github.com/tgstation/-tg-station/pull/14981

**Comment by TheDZD** [comment]
Honestly, it might just be better for us to make a full switch to 510 at some point in the near future. It's still a beta release as of right now, but there are certainly some incentives to doing so (such as the bug where BYOND would generate "fake" lag).

**Comment by Tastyfish** [comment]
Yeah, it seems that everyone else is gearing up for it.

**Comment by Fox-McCloud** [comment]
@Tastyfish 

I'm inclined to agree with @TheDZD here. Would you like to use this PR to do that, or utilize a new PR for said purposes? (not that you should feel obliged that you have to be the one to do it).

**Comment by tigercat2000** [comment]
@Fox-McCloud we should probably make a 510 branch, because it won't be backwards compatible with <509 clients

**Comment by Tastyfish** [comment]
@Fox-McCloud An actual 510 compatibility push should probably be a separate PR.

**Comment by Tastyfish** [comment]
Splitting off the 510 stuff for when we just do the proper 510 switch in the distant future.

**Pull request merged**